### PR TITLE
stm32f1 gpio AF remap ll function without AFIO_MAPR_SWJ_CFG

### PR DESCRIPTION
**Pull request by FRASTM**
To remap or un-remap the different LL_GPIO_AF_EnableRemap_xxx
or LL_GPIO_AF_DisableRemap_xxx does not need to change the
SWJ_CFG bits of the MAPR register.

Signed-off-by: Francois Ramu <francois.ramu@st.com>

### CODE REVIEW

**Comment by FRASTM** [comment]
internal bug: 114125

**Review by erwango — CHANGES_REQUESTED** [review]
Modification could be seen as an API change and I'm not sure it could be accepted in Cube package.
Nacking until we can get a approval from Cube package maintainers.

**Comment by FRASTM** [comment]
Instead change in the STM32CubeF1   API, the modification is done in the zephyr code : https://github.com/zephyrproject-rtos/zephyr/pull/38741

**Pull request closed without merging**
